helm/nginx-ingress-controller-app/values.yaml

```diff
@@ -19,44 +17,18 @@ ingressController:
   legacy: false
   replicas: 3
 
-# configmap contains settings that can be overridden with a custom values
-# configmap.
+# for all the nginx configmap config options see https://github.com/kubernetes/ingress-nginx/blob/master/docs/user-guide/nginx-configuration/configmap.md#configmaps
 configmap:
-  disable-access-log: "false"
-  enable-vts-status: "true"
   error-log-level: "error"
+  # Disables setting a 'Strict-Transport-Security' header, which can be harmful.
+  # See https://github.com/kubernetes/ingress-nginx/issues/549#issuecomment-291894246
   hsts: "false"
+  # Increase hash table size to allow more server names for stability reasons
   server-name-hash-bucket-size: "1024"
-  server-name-hash-max-size: "1024"
   server-tokens: "false"
   worker-processes: "4"
   use-forwarded-headers: "true"
 
-  # optional settings that can be set.
```

Comment:
This is no longer needed, since all configmap settings can be overriden with custom values, no need to add explicit support for each of them.